cmd/api-errors.go

```diff
@@ -632,6 +633,11 @@ var errorCodeResponse = map[APIErrorCode]APIError{
 		Description:    "Configuration update failed because server quorum was not met",
 		HTTPStatusCode: http.StatusServiceUnavailable,
 	},
+	ErrAdminCredentialsMismatch: {
+		Code:           "XMinioAdminCredentialsMismatch",
+		Description:    "Credentials in config mismatch with server env vars",
```

Comment:
s/env vars/environmental variables

I suggested using the expanded form since this is part of the descriptive error message.

Reply:
done.